### DISCUSSION TITLE
QSPI flash :  unrecognized

### DISCUSSION THREAD

**Discussion by yeliangchen2** [polarfire-soc/polarfire-soc-documentation#306] [Linux drivers for PolarFire Soc]
64MB MT25QL512ABB8ESF-0SIT
![image](https://user-images.githubusercontent.com/16462894/230858849-3289078c-7329-4cb5-9631-2f16b502eb31.png)

**Comment by ConchuOD** [comment]
Hey!
Could you please provide some more info on your setup?

Is this an Icicle or some other dev board? If it's a Microchip dev board, are you using one of our releases?
If so, which one?
What QSPI flash are you using?
